config/config.yaml

```diff
@@ -127,8 +127,19 @@ default:
                 handlers:
```

Comment:
Let's drop the config updates from the PR. We want to leave the defaults alone unless we explicitly need to update it.